### DISCUSSION TITLE
Accessing Mojolicious 9.35 from cpan?

### DISCUSSION THREAD

**Discussion by daleif** [mojolicious/mojo#2121] [General]
If 9.35 is currently the official release, why is only 9.34 available on CPAN?

9.34 for example has Mojo::Server that uses `do` which we discussed earlier. 

Right not my cpanfile for cpanm fails on `requires 'Mojolicious', '>=9.35'` (and true enough CPAN only lists 9.34)

Is there a better repro to use?

**Comment by Grinnz** [comment]
Not sure where you got the idea that "9.35 is currently the official release". 9.34 is the latest released version, as you found.

**Reply by daleif** [reply]
(1) https://docs.mojolicious.org/, (2) 9.34 has the bug mentioned in ##2110 which is fixed in 9.35. So why is that version not released.

**Reply by christopherraa** [reply]
Yeah, there seems to be a mismatch between what is "advertised" on docs.mojolicious.org and what is available on CPAN. Docs-site for reference now:

![2023-12-31-125954_516x259_scrot](https://github.com/mojolicious/mojo/assets/924763/3eb105f1-1fdb-4b0b-b16c-88c8bd2e4008)

While version information on CPAN has v9.35 as the latest:

![2023-12-31-130144_777x275_scrot](https://github.com/mojolicious/mojo/assets/924763/3240b943-9b23-48c9-b714-337f0c785656)

This is probably just a matter of version information having been bumped in the source, but no new release being cut and pushed to CPAN.
